draft-ietf-quic-version-negotiation.md

```diff
@@ -392,13 +392,32 @@ its length is not divisible by four), then the endpoint MUST close the
 connection; if the connection was using QUIC version 1, that connection closure
 MUST use a transport error of type TRANSPORT_PARAMETER_ERROR. If an endpoint
 receives a Chosen Version equal to zero, or any Available Version equal to zero,
-it MUST treat it as a parsing failure.
+it MUST treat it as a parsing failure. If a server receives a Version
+Information where the Chosen Version is not included in Available Versions, it
+MUST treat it as a parsing failure.
```

Comment:
What's the purpose of this requirement? Is it just a correctness check?

Reply:
Yes, it makes sure that the server validates that the client follows the MUST.

Reply:
I'm sorry, I don't follow this at all.

Reply:
In `Client-Sent Available Versions` we have `Note that the version in the Chosen Version field MUST be included in this list`. The text we're commenting on here asks the server to validate that the client followed that MUST